common/shareddb.cpp

```diff
@@ -2075,6 +2073,8 @@ void SharedDatabase::LoadLootTables(void *data, uint32 size) {
 			  loottable.mincash,
 			  loottable.maxcash,
 			  loottable.avgcoin,
+			  COALESCE(loottable.content_flags, ''),
```

Comment:
This should be added at the end, not the middle

Also should not be transforming the content; just safely check for it in the row iterator and return an empty string if its null or empty. Content flags can be comma separated in the fields to have multiple so we need to be able to pass all of them